### PR TITLE
Includes SharePoint 2019 as the latest release

### PR DESCRIPTION
**Pull request by iamguptarishi**
The article was mentioning that the latest release for SharePoint is SP 2016 which is not true. Updated to SharePoint 2019

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/upgrade-from-sharepoint-2010.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ Begin by comparing hardware and software requirements. If you don't meet basic r
     
 ### Create a SharePoint hybrid solution between SharePoint Online and SharePoint Server on-premises
 
-Another option (one that may be the best of both on-premises and online worlds for some migration needs) is a hybrid, you can connect SharePoint Server 2013 or 2016 farms to SharePoint Online to create a SharePoint hybrid: [Learn about SharePoint hybrid solutions](https://support.office.com/article/4c89a95a-a58c-4fc1-974a-389d4f195383.aspx).
+Another option (one that may be the best of both on-premises and online worlds for some migration needs) is a hybrid, you can connect SharePoint Server 2013 or 2016 or 2019 farms to SharePoint Online to create a SharePoint hybrid: [Learn about SharePoint hybrid solutions](https://support.office.com/article/4c89a95a-a58c-4fc1-974a-389d4f195383.aspx).
```
Comment (JohanFreelancer9) [inline Enterprise/upgrade-from-sharepoint-2010.md]:
```suggestion
Another option (one that may be the best of both on-premises and online worlds for some migration needs) is a hybrid; you can connect SharePoint Server 2013, Server 2016, or Server 2019 farms to SharePoint Online to create a SharePoint hybrid: [Learn about SharePoint hybrid solutions](https://support.office.com/article/4c89a95a-a58c-4fc1-974a-389d4f195383.aspx).
```

File: Enterprise/upgrade-from-sharepoint-2010.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ If you've decided to create a new Office 365 site, and will manually migrate dat
   
 ### Upgrade SharePoint Server on-premises
 
-As of the latest version of the SharePoint on-premises product (SharePoint Server 2016), SharePoint Server upgrades must go  *serially*, that means there is no way to upgrade from SharePoint Server 2010 to SharePoint Server 2016, directly. 
+As of the latest version of the SharePoint on-premises product (SharePoint Server 2019), SharePoint Server upgrades must go  *serially*, that means there is no way to upgrade from SharePoint Server 2010 to SharePoint Server 2016 or to SharePoint 2019, directly.
```
Comment (JohanFreelancer9) [inline Enterprise/upgrade-from-sharepoint-2010.md]:
```suggestion
As of the latest version of the SharePoint on-premises product (SharePoint Server 2019), SharePoint Server upgrades must go  *serially*, that means there is no way to upgrade from SharePoint Server 2010 to SharePoint Server 2016 or to SharePoint Server 2019 directly. 
```

**Comment by mypil** [comment]
@iamguptarishi - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@iamguptarishi - We would like to follow up. Please assist to merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made.

**Comment by mypil** [comment]
@iamguptarishi - Please assist to merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made.

**Comment by mypil** [comment]
@MSFTTracyP - Please assist to review and merge the content of this PR. Thank you.

**Pull request merged**